### ISSUE TITLE
Unclear duplicate key character map entries lose in quartzkey.py

### ISSUE THREAD

**Issue by pushfoo** [pyglet/pyglet#1096]
TL;DR: Repeated entries in dict literals lose info

## What?

Only the second value will be persisted in the final dict.
https://github.com/pyglet/pyglet/blob/9fe927d1322f2e845288c9b10a54f1ded12b4d08/pyglet/libs/darwin/quartzkey.py#L196-L197

https://github.com/pyglet/pyglet/blob/9fe927d1322f2e845288c9b10a54f1ded12b4d08/pyglet/libs/darwin/quartzkey.py#L232-L233

These are marked as synonyms as synonyms in `key.py` so the duplication may be intended as a note for readers:
https://github.com/pyglet/pyglet/blob/9fe927d1322f2e845288c9b10a54f1ded12b4d08/pyglet/window/key.py#L330-L331

https://github.com/pyglet/pyglet/blob/9fe927d1322f2e845288c9b10a54f1ded12b4d08/pyglet/window/key.py#L366-L367

I came across this while looking at `pyglet/lib` for #1084

**Comment by drivingmenuts** [comment]
That symbol has multiple names in English (HASH, POUND, NUMBER SIGN) depending on usage. Why? Because English likes to mug other languages in dark alleys to go through their pockets for words. The things we do to language are sometimes considered evil in other locales (eg., France)

**Comment by cdeil** [comment]
How to resolve this ticket? Leave as-is or remove the duplicate dict keys that get overwritten by the entry on the next line anyways?

**Comment by benmoran56** [comment]
I'm inclined to just leave them there, if only for reference. I believe that's why they're there in the first place.

**Comment by caffeinepills** [comment]
Closing as Benjamin wants to leave them.
